sci-libs/gdal/gdal-2.0.2-r1.ebuild

```diff
@@ -0,0 +1,321 @@
+
+	# the second sed expression should fix bug 371075
+	sed -i \
+		-e "s:setup.py install:setup.py install --root=\$(DESTDIR):" \
+		-e "s:--prefix=\$(DESTDIR):--prefix=:" \
+		"${S}"/swig/python/GNUmakefile || die
+
+	# -soname is only accepted by GNU ld/ELF
+	[[ ${CHOST} == *-darwin* ]] \
+		&& epatch "${FILESDIR}"/${PN}-1.5.0-install_name.patch \
+		|| epatch "${FILESDIR}"/${PN}-1.5.0-soname.patch
+
+	# Fix spatialite/sqlite include issue
+	sed -i \
+		-e 's:spatialite/sqlite3.h:sqlite3.h:g' \
+		ogr/ogrsf_frmts/sqlite/ogr_sqlite.h || die
+
+	# Fix freexl configure check
+	sed -i \
+		-e 's:FREEXL_LIBS=missing):FREEXL_LIBS=missing,-lm):g' \
+		configure.in || die
+
+	sed \
+		-e "s: /usr/: \"${EPREFIX}\"/usr/:g" \
+		-i configure.in || die
+
+	sed \
+		-e 's:^ar:$(AR):g' \
+		-i ogr/ogrsf_frmts/sdts/install-libs.sh || die
+
+	tc-export AR RANLIB
+
+	eautoreconf
+
+	prepare_python() {
+		mkdir -p "${BUILD_DIR}" || die
+		find "${S}" -type d -maxdepth 1 -exec ln -s {} "${BUILD_DIR}"/ \; ||die
+		find "${S}" -type f -maxdepth 1 -exec cp --target="${BUILD_DIR}"/ {} + ||die
+#		mkdir -p "${BUILD_DIR}"/swig/python || die
+#		mkdir -p "${BUILD_DIR}"/apps || die
+#		cp -dpR --target="${BUILD_DIR}"/swig/ \
+#			"${S}"/swig/{python,SWIGmake.base,GNUmakefile} || die
+#		ln -s "${S}"/swig/include "${BUILD_DIR}"/swig/ || die
+#		ln -s "${S}"/apps/gdal-config "${BUILD_DIR}"/apps/ || die
+#		ln -s "${S}"/port "${BUILD_DIR}"/ || die
+	}
+	if use python; then
+		python_foreach_impl prepare_python
+	fi
+}
+
+gdal_src_configure() {
+	local myopts=""
+
+	if use java; then
+		myopts+="
+			--with-java=$(java-config --jdk-home 2>/dev/null)
+			$(use_with mdb)"
+	else
+		myopts+=" --without-java --without-mdb"
+		use mdb && ewarn "mdb requires java use enabled. disabling"
+	fi
+
+	if [[ -n $use_python ]]; then
+		myopts+=" --with-python"
+	else
+		myopts+=" --without-python"
+	fi
+
+	if use sqlite; then
+		myopts+=" LIBS=-lsqlite3"
+	fi
+
+	# pcidsk is internal, because there is no such library yet released
+	#     also that thing is developed by the gdal people
+	# kakadu, mrsid jp2mrsid - another jpeg2k stuff, ignore
+	# bsb - legal issues
+	# oracle - disabled, i dont have and can't test
+	# ingres - same story as oracle oci
+	# podofo - we use poppler instead they are exclusive for each other
+	# tiff is a hard dep
+	ECONF_SOURCE="${S}" econf \
+		--includedir="${EPREFIX}/usr/include/${PN}" \
+		--disable-static \
+		--enable-shared \
+		--with-expat \
+		--with-geotiff \
+		--with-grib \
+		--with-libtiff \
+		--with-libz="${EPREFIX}/usr/" \
+		--with-ogr \
+		--without-bsb \
+		--without-dods-root \
+		--without-dwgdirect \
+		--without-epsilon \
+		--without-fme \
+		--without-grass \
+		--without-hdf4 \
+		--without-idb \
+		--without-ingres \
+		--without-jp2mrsid \
+		--without-kakadu \
+		--without-libtool \
+		--without-mrsid \
+		--without-msg \
+		--without-oci \
+		--without-pcraster \
+		--without-podofo \
+		--without-sde \
```

Comment:
…and just put `--without-python` here. The configure doesn't really care about that switch, only to determine some random useless magic wrt Python and setuptools that actually does more harm than good.